tests/php/unit/Configuration/ConfigurationTest.php

```diff
@@ -71,14 +72,53 @@ public function testBlacklistFilterInvalid() {
 			new \Yoast_ACF_Analysis_String_Store()
 		);
 
-		Filters\expectApplied( \Yoast_ACF_Analysis_Facade::get_filter_name( 'blacklist' ) )
+		Filters\expectApplied( \Yoast_ACF_Analysis_Facade::get_filter_name( 'blacklist_type' ) )
 			->once()
 			->with( $store )
 			->andReturn( '' );
 
-		$this->assertSame( $store, $configuration->get_blacklist() );
+		$this->assertSame( $store, $configuration->get_blacklist_type() );
+	}
+
+	public function testBlacklistNameFilter() {
+
+		$blacklist_name = new \Yoast_ACF_Analysis_String_Store();
+
+		$configuration = new \Yoast_ACF_Analysis_Configuration(
+			$blacklist_name,
+			new \Yoast_ACF_Analysis_String_Store()
+		);
+
+		$blacklist_name2 = new \Yoast_ACF_Analysis_String_Store();
+
+		$configuration->get_blacklist_name();
+
+		Filters\expectApplied( \Yoast_ACF_Analysis_Facade::get_filter_name( 'blacklist_name' ) )
+			->once()
+			->with( $blacklist_name )
+			->andReturn( $blacklist_name2 );
+
```

Comment:
So just to explain what this does:
It tests that:
- The filter with the name of `\Yoast_ACF_Analysis_Facade::get_filter_name( 'blacklist_name' )` ...
- ... is called once ...
- ... and gets the value of `$blacklist_name` passed ...
- ... and makes the filter return `$blacklist_name2`

Now you can also see why the test failed: As you created a _new_ instance of `\Yoast_ACF_Analysis_String_Store` in the constructor of `\Yoast_ACF_Analysis_Configuration` but expected the filter to get `$blacklist_name` passed (`->with( $blacklist_name )`) the test failed.